code/modules/food_and_drinks/food/foods/sandwiches.dm

```diff
@@ -3,7 +3,24 @@
 //		Burgers		//
 //////////////////////
 
-/obj/item/reagent_containers/food/snacks/brainburger
+// Abstract object used for inheritance. Should never spawn. Needed to not break recipes that use plain burgers; recipes that use "burger" would accept any burger and transfer reagents otherwise.
+
+/obj/item/reagent_containers/food/snacks/burger
+	name = "burger"
+	desc = "If you got this, something broke! Contact a coder if this somehow spawns."
+	icon = 'icons/obj/food/burgerbread.dmi'
+	icon_state = "burger"
+
+/obj/item/reagent_containers/food/snacks/burger/plain
+	name = "burger"
+	desc = "The cornerstone of every nutritious breakfast."
+	icon_state = "hburger"
+	filling_color = "#D63C3C"
+	bitesize = 3
+	list_reagents = list("nutriment" = 6, "vitamin" = 1)
+	tastes = list("bun" = 4, "meat" = 1, "the jungle" = 1)
```

Comment:
Tasting like the jungle seems like something that carried over from monkeyburgers? Not sure what happened there honestly

Reply:
Yeaah. That's actually exactly what happened lol.